smartseq2_single_sample/ss2_single_sample_kco.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
If this is an example input json I think the file name should include input in the name. For example `PairedSingleSampleWf_template.json` is a json we have uploaded in another repo. This one could be `Ss2KcoSingleSample_inputs.json`